packages/core/src/app/shipping/ShippingAddress.spec.tsx

```diff
@@ -1,5 +1,5 @@
 import { CheckoutService, createCheckoutService } from '@bigcommerce/checkout-sdk';
-import { mount, ReactWrapper, shallow } from 'enzyme';
+import { mount, ReactWrapper } from 'enzyme';
```

Comment:
As far as I know we have to update tests with switching to `@testing-library` if we make any changes in oldest one. Because there is no reason to support the old approach if we need to move to a new one.

Reply:
I just removed 'shallow' because it wasn't used anywhere

Reply:
My new added tests doesn't use any specified to enzyme or rts methods